tutorial/01-intro.md

```diff
@@ -4,22 +4,22 @@ ms.localizationpriority: medium
 
 <!-- markdownlint-disable MD002 MD041 -->
 
-This tutorial teaches you how to build a .NET Core console app that uses the Microsoft Graph API to retrieve calendar information for a user.
+This tutorial teaches you how to build a .NET Core console app that uses the Microsoft Graph API.
 
 > [!TIP]
 > If you prefer to just download the completed tutorial, you can download or clone the [GitHub repository](https://github.com/microsoftgraph/msgraph-training-dotnet-core).
 
 ## Prerequisites
 
-Before you start this tutorial, you should have the [.NET Core SDK](https://dotnet.microsoft.com/download) installed on your development machine. If you do not have the SDK or Maven, visit the previous link for download options.
+Before you start this tutorial, you should have the [.NET SDK](https://dotnet.microsoft.com/download) installed on your development machine. If you do not have the SDK or Maven, visit the previous link for download options.
```

Comment:
The way it shows currently is how I've always done tutorials. Here's why I did it that way (not saying I'm right, just giving my reasoning):

- It's not accurate to say .NET SDK version 6.0.102 is *required*. This is likely going to work with any version 6.x.
- I then in a note clarify the specific version it was written with, in case there are some compat issues.
- I don't want folks to think they have to locate and download a specific version.

Does that make sense?